templates/simple/src/bin/dune

```diff
@@ -0,0 +1,4 @@
+(executable
+ (name main)
+ (public_name package)
```

Comment:
very confusing to call everything `package`. Can we come up with a bit better names?

Reply:
Sure, what about just `example`?

Reply:
we could do a library and no executable?

Reply:
I kept both but renamed it to example, if that's okay with you it should be good to go now